airsonic-main/src/main/webapp/WEB-INF/jsp/playQueue.jsp

```diff
@@ -447,11 +469,24 @@
             if ($("#trackNumber" + id)) {
                 $("#trackNumber" + id).text(song.trackNumber);
             }
-            if (song.starred) {
-                $("#starSong" + id).attr("src", "<spring:theme code='ratingOnImage'/>");
+
+            if (!internetRadioEnabled) {
+                // Show star/remove buttons in all cases...
+                $("#starSong" + id).css("display", "");
```

Comment:
What about something like `$("#starSong" + id).show();` instead?

Reply:
Yup, it does work better! Changed.